src/wallet/rpcwallet.cpp

```diff
@@ -440,7 +440,8 @@ static RPCHelpMan sendtoaddress()
                     {"subtractfeefromamount", RPCArg::Type::BOOL, /* default */ "false", "The fee will be deducted from the amount being sent.\n"
                                          "The recipient will receive less bitcoins than you enter in the amount field."},
                     {"replaceable", RPCArg::Type::BOOL, /* default */ "wallet default", "Allow this transaction to be replaced by a transaction with higher fees via BIP 125"},
-                    {"conf_target", RPCArg::Type::NUM, /* default */ "wallet default", "Confirmation target (in blocks), or fee rate (for " + CURRENCY_UNIT + "/kB or " + CURRENCY_ATOM + "/B estimate modes)"},
+                    {"conf_target", RPCArg::Type::NUM, /* default */ "wallet -txconfirmtarget", "Confirmation target (in blocks)\n"
```

Comment:
I see that the third argument here has changed from `"wallet default"` to `"wallet -txconfirmtarget"`, but it's not clear to me what the effect of that would be.

Reply:
This is the RPC help output the user sees. The idea was to make the `conf_target` help correct (some were missing the sat/B mention), consistent and helpful across the 6 RPCs. The txconfirmtarget mention in some of the `conf_target` helps seems more useful info than just "wallet default", so I made them all the same/consistent.

Reply:
This is the RPC help output the user sees. The idea was to make the `conf_target` help correct (some were missing the sat/B mention), consistent and helpful across the 6 RPCs. The txconfirmtarget mention in some of the `conf_target` helps seems more useful info than just "wallet default", so I made them all the same/consistent. Open to disagreement about that :)

Reply:
e.g. before
```
7. conf_target      (numeric, optional, default=wallet default) Confirmation target (in blocks), or fee rate (for BTC/kB or sat/B estimate modes)
```
after
```
7. conf_target      (numeric, optional, default=wallet -txconfirmtarget) Confirmation target (in blocks)
                    or fee rate (for BTC/kB and sat/B estimate modes)
```

Reply:
I see. That looks like a good change to me.